package.json

```diff
@@ -209,6 +210,7 @@
         "url-join": "^4.0.0",
         "url-parse": "^1.5.10",
         "url-slug": "^3.0.2",
+        "webfontloader": "^1.6.28",
```

Comment:
Our distinction between dependencies and devDependencies and arbitrary at best, but roughly, we should put everything that is needed for a build & deploy into `dependencies`.
Since this dependency is needed for a full TS build, it is correct to have it in `dependencies`.